src/main/java/org/desarrolladorslp/technovation/controller/BatchController.java

```diff
@@ -89,6 +90,14 @@ public ResponseEntity registerUserToBatch(@PathVariable String batchId, @PathVar
         return new ResponseEntity(HttpStatus.OK);
     }
 
+    @Secured({"ROLE_ADMINISTRATOR"})
+    @PostMapping("register")
+    public void registerToBatch(@RequestBody RegisterToBatchDTO register){
+        batchService.registerToBatch(register.getRegister(),register.getBatchId());
```

Comment:
I recommend to create a method in the service to encapsulate both methods, otherwise we would be introducing logic in the controller